src/maxdiffusion/configs/base_flux_dev.yml

```diff
@@ -173,7 +173,7 @@ hf_train_files: ''
 hf_access_token: ''
 image_column: 'image'
 caption_column: 'text'
-resolution: 1024
+resolution: 512
```

Comment:
what is the reasoning for changing the resolution from 1024 to 512? Flux default resolution is 1024.